tests/integration/targets/test_target/defaults/main.yml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
Are these "test_target" files placeholders for further work. Not sure what these are for?

Reply:
Didn't know that thanks.